public/app/features/explore/hooks/useExplorePageTitle.ts

```diff
@@ -40,7 +40,7 @@ export function useExplorePageTitle(params: ExploreQueryParams) {
           return Promise.reject();
         }
 
-        return getDatasourceSrv().get(pane.datasource);
+        return getDataSourceSrv().get(pane.datasource);
```

Comment:
It's better to import from runtime but I don't quite get why would it change anything. The one from plugins calls runtime anyway:

```typescript
import { getDataSourceSrv as getDataSourceService } from '@grafana/runtime';

export const getDatasourceSrv = (): DatasourceSrv => {
  return getDataSourceService() as DatasourceSrv;
};
```

Why would it work now 🤔 Is there a race condition that mocks the other import or clears the service?